diff -up exiv2-0.27.2-Source/xmpsdk/CMakeLists.txt.omv~ exiv2-0.27.2-Source/xmpsdk/CMakeLists.txt
--- exiv2-0.27.2-Source/xmpsdk/CMakeLists.txt.omv~	2019-11-17 20:30:17.250470280 +0100
+++ exiv2-0.27.2-Source/xmpsdk/CMakeLists.txt	2019-11-17 20:31:11.457617655 +0100
@@ -56,7 +56,7 @@ if (BUILD_SHARED_LIBS)
 endif()
 
 # 1119  Install libxmp.a for use by third party applications (Thanks, Emmanuel)
-install(TARGETS exiv2-xmp EXPORT exiv2Config
-        LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
-        ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
-)
+#install(TARGETS exiv2-xmp EXPORT exiv2Config
+#        LIBRARY DESTINATION ${CMAKE_INSTALL_LIBDIR}
+#        ARCHIVE DESTINATION ${CMAKE_INSTALL_LIBDIR}
+#)
