diff -up exiv2-0.27.0-Source/samples/CMakeLists.txt.omv~ exiv2-0.27.0-Source/samples/CMakeLists.txt
--- exiv2-0.27.0-Source/samples/CMakeLists.txt.omv~	2018-12-24 01:06:20.116480013 +0100
+++ exiv2-0.27.0-Source/samples/CMakeLists.txt	2018-12-24 01:06:51.914893104 +0100
@@ -90,6 +90,7 @@ endif()
 # ******************************************************************************
 # connection test application
 add_executable(conntest conntest.cpp)
+target_link_libraries(conntest PRIVATE curl)
 list(APPEND APPLICATIONS conntest)
 
 # ******************************************************************************
diff -up exiv2-0.27.0-Source/samples/conntest.cpp.omv~ exiv2-0.27.0-Source/samples/conntest.cpp
--- exiv2-0.27.0-Source/samples/conntest.cpp.omv~	2018-12-24 01:05:46.768049464 +0100
+++ exiv2-0.27.0-Source/samples/conntest.cpp	2018-12-24 01:05:59.775217060 +0100
@@ -3,6 +3,7 @@
 // Tester application for testing the http/https/ftp/ssh/sftp connection
 
 #include <exiv2/exiv2.hpp>
+#include <curl/curl.h>
 
 #include <iostream>
 #include <cstdlib>
